bucket/dbvis.json

```
@@ -36,5 +36,16 @@
     "checkver": {
         "url": "https://www.dbvis.com/download/",
         "regex": "DbVisualizer\\s+([\\d.]+)"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://www.dbvis.com/product_download/dbvis-$version/media/dbvis_windows-x64_$majorVersion_$minorVersion_$patchVersion.zip"
+            },
+            "32bit": {
+                "url": "https://www.dbvis.com/product_download/dbvis-$version/media/dbvis_windows-x32_$majorVersion_$minorVersion_$patchVersion.zip"
+            }
+        },
+        "extract_dir": "DbVisualizer"
```

```suggestion
        }
```

Not dependent on version, so no need to autoupdate